sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/packages.lock.json

```diff
@@ -12,6 +12,12 @@
           "Microsoft.CodeAnalysis.Workspaces.Common": "[1.3.2]"
         }
       },
+      "Newtonsoft.Json": {

```

Comment:
packages.lock.json files will need to be regenerated without this `Newtonsoft.json` reference